pyne/mesh.py

```diff
@@ -1658,3 +1660,46 @@ def step(self, num_steps):
             self.pos = self.size - 1
             at_end = True
         return at_end
+
+def _cell_fracs_sort_vol_frac_reverse(cell_fracs, num_vol_elements):
+    """
+    Sort cell_fracs according to the order of increasing idx and decreasing
+    with vol_frac.
+
+    Parameters
+    ----------
+    cell_fracs : structured array
+            The output from dagmc.discretize_geom(). A sorted, one dimensional
+            array, each entry containing the following fields:
+
+                :idx: int
+                    The volume element index.
+                :cell: int
+                    The geometry cell number.
+                :vol_frac: float
+                    The volume fraction of the cell withing the mesh ve.
+                :rel_error: float
+                    The relative error associated with the volume fraction.
+
+            The array must be sorted with respect to both idx and cell, with
+            cell changing fastest.
+    num_vol_elements: int
+        number of volume elements.
+
+    Returns
+    -------
+    cell_fracs : structured array
+        Sorted cell_fracs.
+    """
+    # sort ascending along idx and vol_frac
+    cell_fracs.sort(order=['idx', 'vol_frac'])
```

Comment:
Another option here might be to temporarily multiply the volume fractions by `-1.0` to simplify this function and still get an effective reversed sort on the volume fracs. This way the entire body of this function would be:

```
cell_fracs['vol_frac'] *= -1.0
cell_fracs.sort(order=['idx', 'vol_frac'])
cell_fracs['vol_frac'] *= -1.0
```

There also might be some specialized method to sort a single field in reverse, but this should work fine for this case I think.

Reply:
Good idea. Suggestion adopted in d3975ab.

Reply:
A few comments about why the values are being modified and replaced here might be nice.